.github/workflows/ocs-operator-ci.yaml

```diff
@@ -108,4 +108,4 @@ jobs:
           check_filenames: true
           check_hidden: true
           skip: vendor
-          ignore_words_list: xdescribe,contails,shouldnot
+          ignore_words_list: xdescribe,contails,shouldnot,colletor
```

Comment:
i think its a typo in the current code base which need to be fixed, we should not add it here